.github/workflows/stale.yml

```diff
@@ -1,20 +1,25 @@
-name: 'Close Stale PRs'
+name: Close Stale PRs
 on:
   schedule:
     - cron: '00 1 * * *'
 
 permissions:
+  contents: write
   pull-requests: write
+  issues: write
 
 jobs:
   close-stale-prs:
     # don't run this workflow on a cron for forks
     if: ${{ github.event_name != 'schedule' || github.repository == 'metabase/metabase' }}
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

Comment:
Are you intentionally switching to the ubuntu-latest in a lot of recent PRs?

Reply:
yes, ones that are just running standard actions